### ISSUE TITLE
lab10 review

### ISSUE THREAD

**Issue by Matteo-Celia** [SalvatorePolito98/Computational_Intelligence24#6]
Hi Salvatore, I will leave just a few suggestions that could improve your method:
First of all, in the Q_table update formula  the term (-learning_rate)*Q_table[state][action] appears twice ( at the beginning of the formula and inside the parenthesis). The one inside the parenthesis can be removed.

The new formula would be: 
Q_table[state][action] = 
(1 - learning_rate) * Q_table[state][action] + learning_rate * (reward +discount_factor *max(Q_table[new_state].values())).
Moreover it could be useful to first train the Q-learning agent and then test it on some amount of games so that the number of wins/losses/draws is computed based on the final version of the Q_table and not in the still-learning version.
